package.json

```
@@ -26,8 +26,7 @@
         "release-firefox": "make browser=firefox type=release",
         "dev-chrome": "make dev browser=chrome type=dev watch=1",
         "beta-firefox": "make beta-firefox browser=firefox type=release",
-        "release-chrome": "make browser=chrome type=release && make chrome-release-zip",
-        "release-update": "git submodule update --remote shared/privacy-configuration/ && npm run generate-etags && node scripts/bumpVersion.js"
+        "release-chrome": "make browser=chrome type=release && make chrome-release-zip"
```

I don't think this change is intentional. I'll revert it.

cc @jonathanKingston 

```suggestion
        "release-chrome": "make browser=chrome type=release && make chrome-release-zip",
        "release-update": "git submodule update --remote shared/privacy-configuration/ && npm run generate-etags && node scripts/bumpVersion.js"
```

---

I think the idea is to only use this in tagging and shouldn't need to be done manually if that's followed.

---

Oh, yeah, I see it now. I'll un-revert it 🙃🙂. Thanks!